### PR TITLE
Add 'zypper' to the valid container.os_packages commands

### PR DESCRIPTION
**Pull request by e4t**

### CODE REVIEW

File: lib/spack/spack/schema/container.py [hunk 1]
```diff
@@ -54,7 +54,7 @@
         "os_packages": {
             "type": "object",
             "properties": {
-                "command": {"type": "string", "enum": ["apt", "yum"]},
+                "command": {"type": "string", "enum": ["apt", "yum", "zypper"]},
```
Comment (alalazo) [inline lib/spack/spack/schema/container.py]:
```suggestion
                "command": {"type": "string", "enum": ["apt", "yum", "zypper", "apk", "yum_amazon"]},
```

**Review by alalazo — CHANGES_REQUESTED** [review]
Thanks for this one, and sorry for the delay reviewing it! We also need to update [this line](https://github.com/spack/spack/blob/11aa2d721e2a7cc0b1ead7687262545e95b17685/lib/spack/docs/containers.rst?plain=1#L619) in the docs.

File: lib/spack/spack/schema/container.py (hunk 1 above)
Comment (e4t) [reply lib/spack/spack/schema/container.py]: You're right. Let me update my pull request.

**Comment by alalazo** [comment]
@spackbot fix style

**Review by alalazo — APPROVED** [review]
Thanks!

**Pull request merged**